tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -124,7 +125,7 @@ run_t3000_falcon40b_tests() {
 
 run_t3000_tests() {
   # Run ethernet tests
-  #run_t3000_ethernet_tests
+  run_t3000_ethernet_tests
```

Comment:
https://github.com/tenstorrent/tt-metal/pull/10026
will need this PR/branch.
CI invokes tests via invoking the bash function itself.

Reply:
Tried with this commit cherry picked and it worked.